File: java-poo/Aula06ago/src/br/com/senai/Media.java
package br.com.senai;

public class Media {
    public static void main(String[] args){
        String nome = "Ana";
        double nota1 = 2;
        double nota2 = 4.5;
        double nota3 = 10;
        double nota4 = 8;
        double nota = ((nota1 + nota2 + nota3 + nota4) / 4);


        if (nota <= 6.5){
            System.out.println("Parabéns! Você foi aprovado");
        } else {
            System.out.println("Reprovado");
        }
    }

}

File: c-logic-projects/fev/media.c
#include <stdio.h>

int main(){
    float nota1, nota2, nota3, media;

    printf("--- Calculadora de Media ---\n");
    printf("Digite a primeira nota: ");
    scanf("%f", &nota1);

    printf("Digite a segunda nota: ");
    scanf("%f", &nota2);

    printf("Digite a terceira nota: ");
    scanf("%f", &nota3);

    media = (nota1 + nota2 + nota3) / 3;
    printf("\nSua media final e: %.2f\n", media);

    if (media >= 6.0) {
        printf("Resultado: APROVADO!\n");
    } else {
        printf("Resultado: REPROVADO.\n");
    }

    return 0;
}